paddle/scripts/paddle_build.bat

```diff
@@ -650,6 +650,7 @@ set /p PADDLE_WHL_FILE_WIN=< whl_file.txt
 pip uninstall -y paddlepaddle
 pip uninstall -y paddlepaddle-gpu
 pip install %PADDLE_WHL_FILE_WIN%
+%PYTHON_ROOT%\python.exe -m pip install %PADDLE_WHL_FILE_WIN%
```

Comment:
seems versions both are 0.0.0 and it will not take effect.
```
2023-06-28 22:07:32 (python_venv) C:\home\workspace\Paddle\build>C:\Python37\python.exe -m pip install C:\home\workspace\Paddle\build\python\dist\paddlepaddle_gpu-0.0.0-cp37-cp37m-win_amd64.whl 
2023-06-28 22:07:33 Processing c:\home\workspace\paddle\build\python\dist\paddlepaddle_gpu-0.0.0-cp37-cp37m-win_amd64.whl
2023-06-28 22:07:34 Collecting httpx (from paddlepaddle-gpu==0.0.0)
2023-06-28 22:07:34   Using cached httpx-0.24.1-py3-none-any.whl (75 kB)
2023-06-28 22:07:34 Requirement already satisfied: numpy>=1.13 in c:\users\administrator\appdata\roaming\python\python37\site-packages (from paddlepaddle-gpu==0.0.0) (1.21.5)
2023-06-28 22:07:34 Requirement already satisfied: Pillow in c:\users\administrator\appdata\roaming\python\python37\site-packages (from paddlepaddle-gpu==0.0.0) (8.3.1)
2023-06-28 22:07:34 Requirement already satisfied: decorator in c:\users\administrator\appdata\roaming\python\python37\site-packages (from paddlepaddle-gpu==0.0.0) (5.0.9)
2023-06-28 22:07:34 Requirement already satisfied: astor in c:\users\administrator\appdata\roaming\python\python37\site-packages (from paddlepaddle-gpu==0.0.0) (0.8.1)
2023-06-28 22:07:34 Requirement already satisfied: paddle-bfloat==0.1.7 in c:\users\administrator\appdata\roaming\python\python37\site-packages (from paddlepaddle-gpu==0.0.0) (0.1.7)
2023-06-28 22:07:34 Requirement already satisfied: opt-einsum==3.3.0 in c:\users\administrator\appdata\roaming\python\python37\site-packages (from paddlepaddle-gpu==0.0.0) (3.3.0)
2023-06-28 22:07:35 Collecting protobuf<=3.20.2,>=3.1.0 (from paddlepaddle-gpu==0.0.0)
2023-06-28 22:07:35   Using cached protobuf-3.20.2-cp37-cp37m-win_amd64.whl (905 kB)
2023-06-28 22:07:35 Requirement already satisfied: certifi in c:\users\administrator\appdata\roaming\python\python37\site-packages (from httpx->paddlepaddle-gpu==0.0.0) (2021.5.30)
2023-06-28 22:07:35 Collecting httpcore<0.18.0,>=0.15.0 (from httpx->paddlepaddle-gpu==0.0.0)
2023-06-28 22:07:35   Using cached httpcore-0.17.2-py3-none-any.whl (72 kB)
2023-06-28 22:07:35 Requirement already satisfied: idna in c:\users\administrator\appdata\roaming\python\python37\site-packages (from httpx->paddlepaddle-gpu==0.0.0) (3.2)
2023-06-28 22:07:35 Collecting sniffio (from httpx->paddlepaddle-gpu==0.0.0)
2023-06-28 22:07:35   Using cached sniffio-1.3.0-py3-none-any.whl (10 kB)
2023-06-28 22:07:35 Collecting h11<0.15,>=0.13 (from httpcore<0.18.0,>=0.15.0->httpx->paddlepaddle-gpu==0.0.0)
2023-06-28 22:07:35   Using cached h11-0.14.0-py3-none-any.whl (58 kB)
2023-06-28 22:07:35 Collecting anyio<5.0,>=3.0 (from httpcore<0.18.0,>=0.15.0->httpx->paddlepaddle-gpu==0.0.0)
2023-06-28 22:07:35   Using cached anyio-3.7.0-py3-none-any.whl (80 kB)
2023-06-28 22:07:35 Collecting exceptiongroup (from anyio<5.0,>=3.0->httpcore<0.18.0,>=0.15.0->httpx->paddlepaddle-gpu==0.0.0)
2023-06-28 22:07:35   Using cached exceptiongroup-1.1.1-py3-none-any.whl (14 kB)
2023-06-28 22:07:35 Requirement already satisfied: typing-extensions in c:\users\administrator\appdata\roaming\python\python37\site-packages (from anyio<5.0,>=3.0->httpcore<0.18.0,>=0.15.0->httpx->paddlepaddle-gpu==0.0.0) (******)
2023-06-28 22:07:35 paddlepaddle-gpu is already installed with the same version as the provided wheel. Use --force-reinstall to force an installation of the wheel.
2023-06-28 22:07:36 Installing collected packages: sniffio, protobuf, h11, exceptiongroup, anyio, httpcore, httpx
2023-06-28 22:07:36   Attempting uninstall: protobuf
2023-06-28 22:07:36     Found existing installation: protobuf 3.20.3
2023-06-28 22:07:36     Uninstalling protobuf-3.20.3:
2023-06-28 22:07:36       Successfully uninstalled protobuf-3.20.3
2023-06-28 22:07:37 ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
2023-06-28 22:07:37 paddlepaddle-gpu 0.0.0 requires protobuf<=3.20.0,>=3.1.0, but you have protobuf 3.20.2 which is incompatible.
2023-06-28 22:07:37 Successfully installed anyio-3.7.0 exceptiongroup-1.1.1 h11-0.14.0 httpcore-0.17.2 httpx-0.24.1 protobuf-3.20.2 sniffio-1.3.0
```

suggest to remove the old one first.

```suggestion
%PYTHON_ROOT%\python.exe -m pip uninstall -y paddlepaddle
%PYTHON_ROOT%\python.exe -m pip uninstall -y paddlepaddle-gpu
%PYTHON_ROOT%\python.exe -m pip install %PADDLE_WHL_FILE_WIN%
```